src/_base/harness/attributes/common.yml

```diff
@@ -99,8 +98,11 @@ attributes.default:
     global:
       conf: []
     # used to limit what is copied into an Nginx static-built image
+    # vendor directory needed for vendor/*/*/*/public/ resources (images etc.)
     copy_directories:
      - = @('app.web_directory')
+     - = @('app.src_resources_directory')
+     - = @('app.vendor_directory')
```

Comment:
this is an awfully large folder, so doesn't solve the project's ticket about size, but I think we can still iterate over it later to just get the public dirs so ok for now